index.html

```diff
@@ -26,6 +26,159 @@ <h1 class="career">I'm a software developer</h1>
+                    <h1 class="multi-post-title">Multi-Post Stories Gain+Glory</h1>
+                    <section class="buttons">
+                        <button class="languages">Ruby on rails</button>
+                        <button class="languages">css</button>
+                        <button class="languages">JavScript</button>
+                        <button class="languages">html</button>
+                    </section>
+                    <button class="see-project">See Project</button>
+                </section>
+            </div>
+        </section>
+        <section class="container">
+            <img class="image-grey" src="/Backgroud-images/Img_Plaholder.png" alt="image">
+            <div class="multi">
+                <section class="multi-post">
+                    <h1 class="multi-post-title">Multi-Post Stories Gain+Glory</h1>
+                    <section class="buttons">
+                        <button class="languages">Ruby on rails</button>
+                        <button class="languages">css</button>
+                        <button class="languages">JavScript</button>
+                        <button class="languages">html</button>
+                    </section>
+                    <button class="see-project">See Project</button>
+                </section>
+            </div>
+        </section>
+        <section class="container">
+            <img class="image-grey" src="/Backgroud-images/Img_Plaholder.png" alt="image">
+            <div class="multi">
+                <section class="multi-post">
+                    <h1 class="multi-post-title">Multi-Post Stories Gain+Glory</h1>
+                    <section class="buttons">
+                        <button class="languages">Ruby on rails</button>
+                        <button class="languages">css</button>
+                        <button class="languages">JavScript</button>
+                        <button class="languages">html</button>
+                    </section>
+                    <button class="see-project">See Project</button>
+                </section>
+            </div>
+        </section>
+        <section class="container">
+            <img class="image-grey" src="/Backgroud-images/Img_Plaholder.png" alt="image">
+            <div class="multi">
+                <section class="multi-post">
+                    <h1 class="multi-post-title">Multi-Post Stories Gain+Glory</h1>
+                    <section class="buttons">
+                        <button class="languages">Ruby on rails</button>
+                        <button class="languages">css</button>
+                        <button class="languages">JavScript</button>
+                        <button class="languages">html</button>
+                    </section>
+                    <button class="see-project">See Project</button>
+                </section>
+            </div>
+        </section>
+        <section class="about-me">
```

Comment:
Your aboutme section should be wrapped in a `section` tag to follow a proper structure and hierarchy due to semantic reasons 👀 

    <section>
        // About you info
        // Languages 
    </section>

Kindly make these changes so that we meet the project requirements 📝